homework11/task1_metaclasses/project_hw11_task1/task1.py

```diff
@@ -0,0 +1,33 @@
+"""
+Vasya implemented nonoptimal Enum classes.
+Remove duplications in variables declarations using metaclasses.
+
+from enum import Enum
+class ColorsEnum(Enum):
+    RED = "RED"
+    BLUE = "BLUE"
+    ORANGE = "ORANGE"
+    BLACK = "BLACK"
+class SizesEnum(Enum):
+    XL = "XL"
+    L = "L"
+    M = "M"
+    S = "S"
+    XS = "XS"
+Should become:
+class ColorsEnum(metaclass=SimplifiedEnum):
+    __keys = ("RED", "BLUE", "ORANGE", "BLACK")
+class SizesEnum(metaclass=SimplifiedEnum):
+    __keys = ("XL", "L", "M", "S", "XS")
+assert ColorsEnum.RED == "RED"
+assert SizesEnum.XL == "XL"
+"""
+
+
+class SimplifiedEnum(type):
+    def __new__(cls, future_class_name, future_class_parents, future_class_attr):
+        attrs = [
+            value for key, value in future_class_attr.items() if key.endswith("__keys")
```

Comment:
`key.endswith("__keys")` is is already mangled?  

You can construct a full name, why do you need to iterate over this dict?